src/theme/divider.ts

```diff
@@ -0,0 +1,115 @@
+export default (config: { colors: string[] }) => ({
+  slots: {
+    base: 'flex items-center align-center text-center',
+    border: 'border-gray-200 dark:border-gray-800',
+    container: 'font-medium text-gray-700 dark:text-gray-200 flex',
+    icon: 'flex-shrink-0 w-6 h-6',
```

Comment:
I'd rather keep `w-5 h-5` for the icon to be consistent with other components and set `size="2xs"` on avatar before the `v-bind`.

Reply:
Like here: https://github.com/benjamincanac/ui3/blob/dev/src/runtime/components/NavigationMenu.vue#L65